src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/ValidateFlowAction.java

```diff
@@ -133,4 +133,14 @@ public ValidateFlowAction(PersistenceManager persistenceManager, FlowOperationsD
     protected String getGenericErrorMessage() {
         return "Could not update flow";
     }
+
+    @Override
+    protected void handleError(FlowUpdateFsm stateMachine, Exception ex, ErrorType errorType, boolean logTraceback) {
+        super.handleError(stateMachine, ex, errorType, logTraceback);
+
+        // Notify about failed validation.
+        stateMachine.getEventListeners().forEach(listener ->
```

Comment:
Please, use "notifyEventListenersOnError" method available in FlowProcessingWithEventSupportFsm class.

Reply:
Fixed.